jekyll/_cci2/config-editor.md

```diff
@@ -67,6 +67,15 @@ If you are not making changes on your main branch, you will need to open a pull
 
 ![Save and run]({{site.baseurl}}/assets/img/docs/config-editor-commit-and-run.png)
 
+## Visual Studio Code extension
+{: #visual-studio-code-extension }
+
+Similar features to the in-app configuration editor can be found in the CircleCI VS Code extension if you would prefer to stay in an editor.
```

Comment:
Not sure if we should specify "editor in your local environment" or something along those lines?

Reply:
Maybe...though I think the context is clear enough by saying it's a VS Code extension - the only place it can be used is in the VS Code editor.